cli/command/service/logs.go

```diff
@@ -92,7 +92,7 @@ func runLogs(dockerCli *command.DockerCli, opts *logsOptions) error {
 		if !client.IsErrServiceNotFound(err) {
 			return err
 		}
-		task, _, err := cli.TaskInspectWithRaw(ctx, opts.target)
```

Comment:
These `err` should be handled (can be in a different PR but..)
cc @dperny I can't find the `moby/moby` PR that does that.